src/components/HighlightedText/styled/index.js

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components';
+import attachThemeAttrs from '../../../styles/helpers/attachThemeAttrs';
+
+export const DefaultHitContainer = attachThemeAttrs(styled.span)`
+    color: ${props => props.palette.text.main};
+    font-weight: bold;
+    font-size: 1rem;
```

Comment:
I think we should add common styles like `font-size` to the component container thus when you pass style or className prop you can override it, look at the first example and try to change font size and it will not change